app/graph/query.rb

```diff
@@ -0,0 +1,6 @@
+Query = GraphQL::ObjectType.define do
```

Comment:
Since this is in `graph/` what are your thoughts on namespacing everything under `Graph::`?

We could also have `Graph::Types::Planet` later.